<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="database_tests.py" name="database_tests.DatabaseTest-20200421172831" skipped="0" tests="13" time="1.314" timestamp="2020-04-21T17:28:32">
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="28" name="test_account_exists_with_existing_account" time="0.101" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="24" name="test_account_exists_with_nonexisting_account" time="0.093" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="35" name="test_add_customer" time="0.097" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="81" name="test_add_order_with_one_order" time="0.147" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="49" name="test_find_credentials_with_non_existing_credentials" time="0.097" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="177" name="test_get_cart_with_no_previous_cart" time="0.092" timestamp="2020-04-21T17:28:31">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="186" name="test_get_cart_with_previous_cart" time="0.092" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="73" name="test_get_categories_with_no_products_in_database" time="0.087" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="67" name="test_get_products_with_no_products_in_database" time="0.111" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="56" name="test_get_role_with_admin_account" time="0.078" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="60" name="test_get_role_with_customer_account" time="0.088" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="109" name="test_update_cart_with_no_previous_cart" time="0.096" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
	<testcase classname="database_tests.DatabaseTest" file="src\unittest\python\database_tests.py" line="137" name="test_update_cart_with_previous_cart" time="0.137" timestamp="2020-04-21T17:28:32">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
